lib/apress/selenium_eti/spec/company_site/eti/product_creation_spec.rb

```diff
@@ -63,7 +63,7 @@
     context 'когда копируем товар' do
       before(:all) do
         @product = {
-          name: Faker::Name.title,
+          name: Faker::Number.number(5),
```

Comment:
`NOTE: Faker::Name.title is deprecated; use Faker::Job.title instead. It will be removed on or after 2018-09-01.`
Но `Faker::Job.title` использовать тоже проблемно, т.к попадаются слова, по которым происходит автопривязка характеристик и не кликается ссылка на открытие попапа привязки портальных характеристик. Учту это в рефакторинге, тут пока пусть в таком виде будет.